gr-blocks/grc/blocks_complex_to_interleaved_short.block.yml

```diff
@@ -22,19 +27,19 @@ inputs:
 
 outputs:
 -   domain: stream
-    dtype: short
+    dtype: ${output_type}
     vlen: ${ vector_output.vlen }
 
 templates:
     imports: from gnuradio import blocks
-    make: blocks.complex_to_interleaved_short(${vector_output},${scale_factor})
+    make: blocks.complex_to_interleaved_short(${output_type == 'sc16' or vector_output},${scale_factor})
     callbacks:
     - set_scale_factor(${scale_factor})
 
 cpp_templates:
     includes: ['#include <gnuradio/blocks/complex_to_interleaved_short.h>']
     declarations: 'blocks::complex_to_interleaved_short::sptr ${id};'
-    make: 'this->${id} = blocks::complex_to_interleaved_short::make(${vector_output});'
+    make: 'this->${id} = blocks::complex_to_interleaved_short::make(${output_type == "sc16" || vector_output});'
```

Comment:
This leads to an syntax error.

Replace || by or and the cpp code will be generated.